scripts/gen-descriptions-ios.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
```

Comment:
Why have we added the ios descriptions generator to the Android repo?

Reply:
Because we removed the descriptions from the iOS repo